docs/source/getting_started/examples/sglang-integration/hicache-integration-v1.md

```diff
@@ -216,25 +218,77 @@ The following parameters control SGLang HiCache behavior when using Mooncake as
   - `direct`: Standard CUDA memory copy operations
   - `kernel`: GPU-assisted I/O kernels (recommended for better performance)
 
-- **`--hicache-mem-layout {layer_first,page_first}`**: Memory layout for the host memory pool:
+- **`--hicache-mem-layout {layer_first,page_first}`**: Memory layout for the host memory pool. `page_first` or `page_first_direct` are required if use Mooncake backend:
```

Comment:
This should be three options, right?

choices=["layer_first", "page_first", "page_first_direct"]

Reply:
updated. Thanks for catching this.